x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/ApiKeyService.java

```diff
@@ -1218,6 +1221,44 @@ public void ensureEnabled() {
         }
     }
 
+    public void crossClusterApiKeyUsageStats(ActionListener<Map<String, Object>> listener) {
+        final SecurityIndexManager frozenSecurityIndex = securityIndex.freeze();
```

Comment:
My intention was to report this usage stats regardless of whether the service is enabled because they seem to be separate concerns to me. But on a second thought, I think it might not make much sense or is useful to report these numbers when the service is disabled. When we disable a realm, the realm's usage stats is not reported either. So I took your suggestion to return an empty map in this case. Thanks!